src/include/cart/swim.h

```diff
@@ -55,7 +55,7 @@ enum swim_member_status {
 struct swim_member_state {
 	uint64_t		 sms_incarnation; /**< incarnation number */
 	enum swim_member_status	 sms_status;	  /**< status of member */
-	uint32_t		 sms_padding;
+	uint32_t		 sms_delay;
```

Comment:
minor: a comment here would help to describe what the delay is

Reply:
done